docs/tutorials/model_contribution.md

```diff
@@ -1,4 +1,4 @@
-# ElasticDL Model Building
+# ElasticDL Model Contribution
```

Comment:
dataset_fn and eval_metrics_fn, the parameters in the signature are not exactly the same with the following statement.